src/NexusMods.App.UI/WorkspaceSystem/NewTabPage/NewTabPageView.axaml

```diff
@@ -38,13 +38,15 @@
                             <DataTemplate DataType="{x:Type workspace:INewTabPageSectionItemViewModel}">
 
                                 <Border Classes="Mid Rounded">
-                                    <navigation:NavigationControl Width="375"
+                                    <navigation:NavigationControl
                                             Height="56"
                                             Padding="0"
+                                            Width="375"
+                                            Background="{StaticResource SurfaceMidBrush}"
```

Comment:
The class should already be setting Mid background, you shouldn't need to set the background explicitly.
Did it not work for some reason?

Reply:
Class `Mid` is for `Border.Mid` only.

Reply:
Ah sorry, I read that wrong. 
I checked and I didn't see a specific button style on figma for these so I guess this is fine then. Might want to make a proper class if we have other places using similar buttons in the future.